tests/source/issue-3343/one.rs

```diff
@@ -0,0 +1,14 @@
+// rustfmt-version: One
```

Comment:
this file `tests/source/issue-3343/one.rs` can be removed since it is the same as `tests/target/issue-3343/one.rs`